vuepress/docs/next/tutorials/create/mfe/angular.md

```diff
@@ -74,245 +88,172 @@ This is the expected output:
+::: tip Yay!
+You now have an Angular micro frontend running.
 :::
 
-If you want to use file names with content hashes to avoid potential caching issues in your browser, you can update the `Custom UI` field of your widget after building new versions of your micro frontend. Widget configuration is covered in the next section.
+## Build the Bundle and Install to Entando
 
-## Host Micro Frontend
+### Build and Publish the Bundle 
 
-Now we're ready to host our micro frontend in Entando.
+1. Open `entando.json` in the bundle root folder and add the following to the `angular-widget` parameters. 
+```json
+"buildFolder": "dist/angular-widget"
 
-### Create Public Folder
-
-1. Navigate to `Entando App Builder` in your browser.
-
-2. Click `Administration` at the lower left hand side of the screen.
-
-3. Click the `File Browser` tab.
-
-2. Click the `public` folder.
-
-4. Click `Create Folder`.
-
-5.  Enter `angular-widget`
-
-7. Click `Save`.
-
-8. Click `angular-widget`.
-
-9. Click 'Upload Files`.
-
-10. Upload the following files from `angular-widget/dist/angular-widget`:
-
-- `main-es2015.js`
-- `polyfills-es2015.js`
-- `runtime-es2015.js`
-
-::: warning Additional Deployment Options
-1. Install the micro frontend from a bundle in the `Local Hub`.
-2. Add the micro frontend to `Entando App Builder`.
-3. Load the micro frontend from an API.
-:::
-
-### Add Widget
-
-1. Go to `Components > Micro frontends & Widgets` in the Entando App Builder.
-2. Click `Add` at the lower right.
-
-![New widget screen](./img/new-widget-screen.png)
-
-3. Enter the following:
-- `Code: angular_widget` → note: dashes are not allowed
-- `Title: Angular Widget` → for both English and Italian languages
-- `Group: Free Access`
-- `Custom UI:`
-
-``` ftl
-<#assign wp=JspTaglibs[ "/aps-core"]>
-<script async src="<@wp.resourceURL />angular-widget/main-es2015.js"></script>
-<script async src="<@wp.resourceURL />angular-widget/polyfills-es2015.js"></script>
-<script async src="<@wp.resourceURL />angular-widget/runtime-es2015.js"></script>
-
-<angular-widget />
 ```
+2. Add a thumbnail JPEG or PNG file to the root bundle folder. It must be named thumbnail and not exceed 100kB, e.g. thumbnail.png
 
-4. Click `Save`.
-
-::: tip
-`<#assign wp=JspTaglibs[ "/aps-core"]>` gives you access to the `@wp` object where you can use environment variables like `resourceURL`.
-:::
-
-### See It in Action
-
-Let's see the Angular micro frontend in action on our page.
-
-#### Add Page
-
-::: warning Note
-If you've already configured your home page:
-
-<ol type="i">
-  <li>Next to the <b>Home</b> folder, under <b>Actions</b>, click <b>Configure</b>.</li>
-  <li>Skip to the <b>Add Widget</b> section.</li>
-</ol>
-:::
-
-Let's add our widget to the `Home` page.
-
-1. Go to `Pages` → `Management`
-
-2. Next to the `Home` folder, under `Actions`, click `Edit`.
-
-3. Next to `Page Template` select `Service Page`.
+3. Build the components
+```sh
+ent bundle build --all
```

Comment:
We can skip this step and go straight to ent bundle pack since it includes a build.